src/Console/Command/CompareCommand.php

```diff
@@ -0,0 +1,355 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Console\Command;
+
+use PhpCsFixer\Config;
+use PhpCsFixer\ConfigInterface;
+use PhpCsFixer\Console\ConfigurationResolver;
+use PhpCsFixer\Fixer\FixerInterface;
+use PhpCsFixer\FixerFactory;
+use PhpCsFixer\RuleSet;
+use PhpCsFixer\RuleSetInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Helper\Table;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * @author Adamo Aerendir Crespi <hello@aerendir.me>
+ */
+final class CompareCommand extends Command
+{
+    const COMMAND_NAME = 'compare';
+    const YES = "<fg=green;>\xE2\x9C\x94</>";
+    const NO = "<fg=red;>\xE2\x9C\x96</>";
+
+    /** @var ConfigInterface $defaultConfig */
+    private $defaultConfig;
+
+    /** @var FixerFactory $fixerFactory */
+    private $fixerFactory;
+
+    /** @var bool $hideConfigured */
+    private $hideConfigured;
+
+    /** @var bool $hideEnabled */
+    private $hideEnabled;
+
+    /** @var bool $hideRisky */
+    private $hideRisky;
+
+    /** @var bool $hideRisky */
+    private $hideInherited;
+
+    /** @var array $builtInFixers */
+    private $builtInFixers;
+
+    /** @var array $configuredFixers */
+    private $configuredFixers;
+
+    /** @var array $enabledFixers */
+    private $enabledFixers;
+
+    /** @var array $riskyFixers */
+    private $riskyFixers;
+
+    /** @var int $countInherited */
+    private $countInherited = 0;
+
+    /** @var array $ruleSets Stores all the RuleSets and the Fixers they have */
+    private $ruleSets = [];
+
+    /** @var array $list */
+    private $list = [];
+
+    /** @var array $dump */
+    private $dump;
+
+    /**
+     * @param null|FixerFactory $fixerFactory
+     */
+    public function __construct(FixerFactory $fixerFactory = null)
+    {
+        parent::__construct();
+
+        if (null === $fixerFactory) {
+            $fixerFactory = new FixerFactory();
+            $fixerFactory->registerBuiltInFixers();
+        }
+
+        $this->fixerFactory = $fixerFactory;
+        $this->defaultConfig = new Config();
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setName(self::COMMAND_NAME)
+            ->setDefinition(
+                [
+                    new InputOption('config', '', InputOption::VALUE_REQUIRED, 'The path to a .php_cs file.'),
```

Comment:
```diff
-.php_cs
+config
```

there could be multiple configs named differently

Reply:
Done